qermit/mock_backend/noisy_aer_backend.py

```diff
@@ -0,0 +1,69 @@
+from pytket.extensions.qiskit import AerBackend  # type: ignore
+from pytket import OpType
+import qiskit.providers.aer.noise as noise  # type: ignore
+
+
+class NoisyAerBackend(AerBackend):
+    """ AerBacked with simple depolarising and SPAM noise model. Depolarising
+    noise is added to the gateset {OpType.CX, OpType.H, OpType.Rz,
+    OpType.Rz, OpType.Measure} and circuits should be rebased into
+    that gateset before running.
+    """
+
+    noisy_gate_set = {OpType.CX, OpType.H, OpType.Rz, OpType.Rz, OpType.Measure}
+
+    def __init__(self, n_qubits: int, prob_1: float, prob_2: float, prob_ro: float):
+        """AerBacked with simple depolarising and SPAM noise model.
+
+        :param n_qubits: The number of qubits available on the backend.
+        :type n_qubits: int
+        :param prob_1: The depolarising noise error rates on single qubit gates.
+        :type prob_1: float
+        :param prob_2: The depolarising noise error rates on two qubit gates.
+        :type prob_2: float
+        :param prob_ro: Error rates of symmetric uncorrelated SPAM errors.
+        :type prob_ro: float
+        """
+
+        noise_model = self.depolarizing_noise_model(n_qubits, prob_1, prob_2, prob_ro)
```

Comment:
I reckon move this `noise_model` definition into the `super().__init__`, or add a type definition for it